### PR TITLE
do it

### PR DESCRIPTION
**Pull request by n2oneProgrammer**

### CODE REVIEW

File: fileSystem/views.py [hunk 1]
```diff
@@ -152,18 +159,19 @@ def get(self, request):
...
     field_name = 'the_file'
 
     def check_permissions(self, request):
         if not fileSystemPerm:
             return Response(status=status.HTTP_403_FORBIDDEN)
 
-class UploadFileComplete(ChunkedUploadCompleteView):
 
+class UploadFileComplete(ChunkedUploadCompleteView):
+    response: str
```
Comment (jaanonim) [inline fileSystem/views.py:174]: We need it?

File: fileSystem/views.py [hunk 2]
```diff
@@ -174,24 +182,26 @@ def check_permissions(self, request):
     def on_completion(self, uploaded_file, request):
         name = uploaded_file.name
         path = settings.MEDIA_ROOT + name
-
-        f = file.objects.create(name=name, upload=path)
+        SaveFile(path, uploaded_file)
```
Comment (jaanonim) [inline fileSystem/views.py:185]: This line save file to the our server storage. Do we want to do it?

File: fileSystem/googleUpload.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
+from __future__ import print_function
```
Comment (jaanonim) [inline fileSystem/googleUpload.py]: Not used

**Review by jaanonim — CHANGES_REQUESTED** [review]
What about downloading these files?

File: fileSystem/views.py (hunk 1 above)
Comment (jaanonim) [reply fileSystem/views.py:174]: I mean ` response: str`

File: fileSystem/views.py (hunk 2 above)
Comment (n2oneProgrammer) [reply fileSystem/views.py:185]: yes, because I send file from disk. It's simple than send by chunks to google.

File: fileSystem/views.py (hunk 1 above)
Comment (n2oneProgrammer) [reply fileSystem/views.py:174]: I know. It increases readability code.

**Comment by n2oneProgrammer** [comment]
> What about downloading these files?

I upload file to shared folder for everyone and  use generated link from google drive

**Review by jaanonim — APPROVED** [review]

**Pull request merged**